### PR TITLE
Fix AddTable for realtime tables

### PR DESCRIPTION
**Pull request by npawar**
Fixing a bug happened as a side effect of https://github.com/apache/pinot/pull/9228

### CODE REVIEW

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/AddTableCommand.java [hunk 1]
```diff
@@ -184,14 +207,38 @@ public boolean execute()
...
+    if (_tableConfigFile != null) {
+      TableConfig tableConfig = attempt(() -> JsonUtils.fileToObject(new File(_tableConfigFile), TableConfig.class),
+          "Failed reading table config " + _tableConfigFile);
+      rawTableName = TableNameBuilder.extractRawTableName(tableConfig.getTableName());
+      TableType tableType = TableNameBuilder.getTableTypeFromTableName(tableConfig.getTableName());
+      if (tableType == TableType.OFFLINE) {
+        offlineTableConfig = tableConfig;
+      } else {
+        realtimeTableConfig = tableConfig;
+      }
```
Comment (walterddr) [inline pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/AddTableCommand.java:222]: so the table is default to REALTIME if no type is extracted out?

**Review by walterddr — APPROVED** [review]
looks good. have one quick question

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/AddTableCommand.java (hunk 1 above)
Comment (walterddr) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/AddTableCommand.java:222]: should we enforce this and throw if table type is unknown?

Comment (npawar) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/AddTableCommand.java]: actually realized that we can just get tableType directly from config, no extraction needed. updated

**Comment by mcvsubbu** [comment]
@npawar can you just add a note on what the bug was? We pull from the master, so we may have pulled the buggy version. It will be useful to know  so we can warn our users (or expect some strange behavior). thanks

**Pull request merged**